src/Angor/Client/Pages/Recover.razor

```diff
@@ -897,6 +956,12 @@
 
             project.RecoveryTransactionId = recoveryTransaction.Transaction.GetHash().ToString();
             project.AmountInRecovery = totalsats;
+                      
+            // Increment investors in recovery count or set to 1 if null
+            project.InvestorsInRecovery = (project.InvestorsInRecovery ?? 0) + 1;
```

Comment:
I am not sure I understand the `InvestorsInRecovery` is it how many other investors are recovering their funds?
If you want to know this data it must come from the indexer no be calcualted.